test/system/appointments_test.rb

```diff
@@ -58,10 +58,10 @@ def select_first_available_date
 
     visit account_home_url
 
-    within row_containing(@held_item1.complete_number) { assert_text "Scheduled for pick-up" }
-    within row_containing(@held_item2.complete_number) { assert_text "Ready for pickup" }
-    within row_containing(@borrowed_item1.complete_number) { assert_text "Scheduled for drop-off" }
-    within row_containing(@borrowed_item2.complete_number) { assert_text "Checked-out" }
+    within(row_containing(@held_item1.complete_number)) { assert_text "Scheduled for pick-up" }
```

Comment:
these assertions weren't being executed with the in-line block syntax, but this change seems to work!

Reply:
Good catch.